cmd/download.go

```diff
@@ -90,7 +100,15 @@ func templatizeImageset(release, folder string) {
 	version := release
 
 	// If we support ACM 2.6, then there should be logic to add ACM as a param to the CLI and then a map to hace ACM to AI SHAs
-	d := ImageSet{channel, version, ACM25AssistedInstallerAgentSHA, ACM25AssistedInstallerSHA}
+	if aiInstallerSha == "" {
+		aiInstallerSha = ACM25AssistedInstallerSHA
```

Comment:
We should probably start renaming ACM25AssistedInstallerSHA by ACMDefaultAssistedInstallerSHA or ACMAssistedInstallerSHA (same for the installer). So, it will have a default based on the ACM hub version.

Reply:
Great suggestion